catalog/src/test/java/it/eng/catalog/model/CatalogErrorTest.java

```diff
@@ -40,17 +45,25 @@ public void testPlain() {
 	}
 	
 	@Test
+	@DisplayName("Verify valid protocol object serialization")
 	public void testPlain_protocol() {
-		String result = Serializer.serializeProtocol(catalogError);
-		assertTrue(result.contains(DSpaceConstants.CONTEXT));
-		assertTrue(result.contains(DSpaceConstants.TYPE));
-		assertTrue(result.contains(DSpaceConstants.DSPACE_CODE));
-		assertTrue(result.contains(DSpaceConstants.DSPACE_REASON));
+		JsonNode result = Serializer.serializeProtocolJsonNode(catalogError);
+		assertNotNull(result.get(DSpaceConstants.CONTEXT).asText());
+		assertNotNull(result.get(DSpaceConstants.TYPE).asText());
+		assertNotNull(result.get(DSpaceConstants.DSPACE_CODE).asText());
+		assertNotNull(result.get(DSpaceConstants.DSPACE_REASON).asText());
 		
 		CatalogError javaObj = Serializer.deserializeProtocol(result, CatalogError.class);
 		validateJavaObj(javaObj);
 	}
 	
+	@Test
+	@DisplayName("Missing @context and @ype")
```

Comment:
Missing t in @type :)

Reply:
Fixed in all tests, since it is copy-paste error